src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/README.md

```diff
@@ -107,7 +114,7 @@ Microsoft::Azure::Batch::SoftwareEntitlement::Cleanup();
 ```
 
 ## Limitations
-The root certificate in the server's SSL certificate chain cannot be referenced by the ```ssl_cert_thumbprint```.
+The root certificate in the server's SSL certificate chain cannot be referenced by the ```ssl_cert_thumbprint``` specified in the call to ```AddSslCertificate```.
```

Comment:
A server's SSL connection will have a chain of certs associated with it, starting with the server's own certificate (a leaf) and then moving up the tree to a trusted root.  The root certificate of the certificate chain is not included in the list of certificates returned by OpenSSL.  So you can't specify the thumbprint/common name of the root certificate in the call to AddSslCertificate and expect it to work.